frontend/packages/akr/src/components/clerkTranslator/listing/ClerkTranslatorListing.tsx

```diff
@@ -56,102 +55,103 @@ const ListingRow = ({ translator }: { translator: ClerkTranslator }) => {
     <TableRow
-      data-testid={`clerk-translators__id-${translator.id}-row`}
-      selected={selected}
+      className="cursor-pointer"
       onClick={handleRowClick}
+      selected={selected}
     >
       <TableCell padding="checkbox">
-        <Checkbox checked={selected} color={Color.Secondary} />
+        <Checkbox
+          data-testid={`clerk-translators__id-${translator.id}-row`}
+          checked={selected}
+          color={Color.Secondary}
+          onClick={handleCheckboxClick}
+        />
       </TableCell>
       <TableCell>
         <Text>{`${lastName} ${firstName}`}</Text>
       </TableCell>
-      <TableCell>
-        <div className="rows">
-          {authorisations.map(({ languagePair: { from, to } }, idx) => (
-            <Text key={idx}>
-              {`${translateLanguage(from)} - ${translateLanguage(to)}`}
-            </Text>
-          ))}
-        </div>
-      </TableCell>
-      <TableCell>
-        <div className="rows">
-          {authorisations.map(({ basis }, idx) => (
-            <Text key={idx}>{basis}</Text>
-          ))}
-        </div>
-      </TableCell>
-      <TableCell>
-        <div className="rows">
-          {authorisations.map(({ termBeginDate }, idx) => (
-            <Text key={idx}>{DateUtils.formatOptionalDate(termBeginDate)}</Text>
-          ))}
-        </div>
-      </TableCell>
-      <TableCell>
-        <div className="rows">
-          {authorisations.map(({ termEndDate }, idx) => (
-            <Text key={idx}>{DateUtils.formatOptionalDate(termEndDate)}</Text>
-          ))}
-        </div>
-      </TableCell>
-      <TableCell>
-        <div className="rows">
-          {authorisations.map((a, idx) => (
-            <Text key={idx}>
-              {AuthorisationUtils.isAuthorisationEffective(a, currentDate)
-                ? translateCommon('yes')
-                : translateCommon('no')}
-            </Text>
-          ))}
-        </div>
-      </TableCell>
-      <TableCell>
-        <div className="rows">
-          {authorisations.map(({ permissionToPublish }, idx) => (
-            <Text key={idx}>
-              {permissionToPublish
-                ? translateCommon('yes')
-                : translateCommon('no')}
-            </Text>
-          ))}
-        </div>
-      </TableCell>
-      <TableCell>
-        <div className="columns gapped-sm">
-          <Button
-            data-testid={`clerk-translators__id-${translator.id}-more-btn`}
-            to={translatorDetailsURL(translator.id)}
-            component={Link}
-            color={Color.Secondary}
-            onClick={handleDetailsBtnClick}
-            endIcon={<ArrowForwardIosOutlinedIcon />}
-          >
-            {t('detailsButton')}
-          </Button>
-        </div>
-      </TableCell>
+      {Object.values(AuthorisationColumn).map((activeColumn) => (
+        <TableCell key={activeColumn}>
+          <div className="rows">
+            {authorisations.map((authorisation, idx) => (
```

Comment:
tää on ehkä henk. koht. preferenssi mutta oon koittanut välttää lyhenteitä lyhyissä termeissä, parin characterin säästön vastapuolella on imo todella hyvin luettavaa tavaraa (idx => index, btn => button jne.). Meillä on ehkä koodissa käytetty vähän molempia mutta jos koodin pitäisi olla kuin helppolukuinen tarina niin pitempi kirjoitustyyli ehkä parempi?

Reply:
joo, mulle sopii molemmat. Jatkossa voitte sopia kummalla haluatte edetä. Mitä mieltä @pkoivisto on?

Reply:
Heh, itse olen taas pitänyt tuollaisista lyhenteistä - silloin kun ne ovat "totuttuja". Esim. tässä oikeastaan `idx` sijaan melkein sanoisin että `i` voisi olla paras valinta, ihan siitä syystä että se on (ainakin minulle) tuttu konventio tässä mäppäyksen kontekstissa. Toinen samanlainen konteksti jossa tutut lyhenteet auttavat on esim. reducen yhteydessä: itse muistan signaturen muodossa `.reduce((acc, val) => ...)`, mutta voi toki olla että tässä vaikkapa vaikutteet muista kielistä läikkyvät js:n puolelle. :)

Reply:
That said, ei mulla asiasta kovin vahvaa mielipidettä ole. :)

Reply:
Joo, voidaan pohtia tätä tän ulkopuolella :)